data/json/monsters.json

```diff
@@ -2405,8 +2405,8 @@
   {
     "id": "mon_tankbot",
     "type": "MONSTER",
-    "name": "tank drone",
-    "description": "The Northrup Emancipator is the first and only automated tank ever produced, made shortly after the split-up of Northrup Grumman.  Clad in depleted uranium plating, and armed with advanced munitions and a 120-mm gun, it is capable of delivering extraordinary firepower.",
+    "name": "Beagle Mini-Tank UFV",
+    "description": "The Northrup Beagle is a tankette-sized urban warfare UGV. Sporting an anti-tank missile launcher, 40mm grenade launcher, and numerous anti-infantry weapons, it's designed for high-risk urban fighting.",
```

Comment:
Needs to go smaller, it's still not tankette-sized as long as it's single tile.

Reply:
UFV is an unmanned flying vehicle.